src/server/connectors/address-lookup/organisationRemovaFromLookup.test.js

```diff
@@ -0,0 +1,303 @@
+const {
+  organisationRemovalFromLookup
+} = require("./organisationRemovalFromLookup");
+
+describe("Function: organisationRemovalFromLookup", () => {
+  beforeEach(() => {
```

Comment:
I tihnk this needs a seperate describe for each case. Will chat about this.

Reply:
so much easier to read now. I think you could have even removed more of the unaffected fields a.g. postcode, county. After the first couole of tests just to make mroe concise but don't bother now as is not important: the tests are clear and comprehensive.